validator-api/src/network_monitor/monitor/sender.rs

```diff
@@ -74,6 +81,8 @@ pub(crate) struct PacketSender {
     // behaviour is unlikely.
     active_gateway_clients: HashMap<[u8; PUBLIC_KEY_LENGTH], GatewayClient>,
 
+    // I guess that will be required later on if credentials are got per gateway
+    // aggregated_verification_key: Arc<VerificationKey>,
```

Comment:
I'd say we should add fields when we actually need them, I'm not really seeing the point in adding it now as commented code